### ISSUE TITLE
Unable to login to store.playstation.com

### ISSUE THREAD

**Issue by BlueRaja** [EFForg/privacybadger#1759]
I'm unable to log into store.playstation.com with this plugin enabled (Firefox version).  With the plugin disabled, login works just fine.

**Comment by ghostwords** [comment]
When you say "disabled", do you mean disabled on the site (click "Disable Privacy Badger for This Site" in the popup on the page), or disabled in `about:addons`?

**Comment by BlueRaja** [comment]
Just tested, doing either one works.

**Comment by BlueRaja** [comment]
With the rollout of the new playstation store today, login now work, but many other parts of the site are still broken.

The issue appears to be that Privacy Badger blocks all the playstation.net subdomains (note that the store is at playstation.com).

static-resource.np.community.playstation.net
apollo2.dl.playstation.net
homer.dl.playstation.net
commerce.api.np.km.playstation.net
eventcom.api.np.km.playstation.net
image.api.np.km.playstation.net
recs.api.np.km.playstation.net

`eventcom` is the only one that could be possibly be considered 'tracking' (it's used for non-user-specific telemetry and error-reporting).  The rest are all CDNs and normal APIs.

Meanwhile mookie1.com, a domain whose only purpose is tracking, defaults to green...

**Comment by ghostwords** [comment]
Could you run the following debugging script (see [here](gist.github.com/anonymous/8aebb8e48a1ce3c256b21dcfe37af84b) for more information) in Badger's background page console and share what gets printed out?
```javascript
(function () {
  const STR = "playstation.net";
  console.log("**** ACTION_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('action_map').getItemClones(), (obj, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(obj, null, 2));
  });
  console.log("**** SNITCH_MAP for", STR);
  _.each(badger.storage.getBadgerStorageObject('snitch_map').getItemClones(), (sites, domain) => {
    if (domain.indexOf(STR) != -1) console.log(domain, JSON.stringify(sites, null, 2));
  });
}());
```
To get to the background page console in Chrome, visit `chrome://extensions`, make sure "Developer mode" is checked, click on the "background page" link in Privacy Badger's row, and select the Console tab.

In Firefox, visit `about:debugging`, enable add-on debugging, click Debug next to Privacy Badger, click the OK button on the popup warning about remote debugging, and enter the above script into the console after the >>.

**Comment by ghostwords** [comment]
@BlueRaja Could you provide the information requested above? Let me know if you have any questions.

**Comment by ghostwords** [comment]
Should be fixed by 30e2f2ae5c52957998e00433ef416d72aae5e424.
